### PR TITLE
[FIX] Update some informations on README

### PR DESCRIPTION
**Pull request by djorkaeffalexandre**
I only fixed an information on README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -346,7 +346,7 @@ Every API returns a Promise but also has a corresponding API with 'Sync' on the
 | [getTotalMemory()](#gettotalmemory)                               | `Promise<number>`   |  ✅  |   ✅    |   ❌    |
 | [getUniqueId()](#getuniqueid)                                     | `string`            |  ✅  |   ✅    |   ✅    |
 | [getUsedMemory()](#getusedmemory)                                 | `Promise<number>`   |  ✅  |   ✅    |   ❌    |
-| [getUserAgent()](#getuseragent)                                   | `Promise<string>`   |  ✅  |   ✅    |   ❌    |
+| [getUserAgent()](#getuseragent)                                   | `string`            |  ✅  |   ✅    |   ❌    |
```
Comment (mikehardy) [inline README.md]: getUserAgent is async and returns a promise https://github.com/react-native-community/react-native-device-info/blob/master/src/index.ts#L363

File: README.md [hunk 2]
```diff
@@ -945,7 +945,7 @@ DeviceInfo.getPreviewSdkInt().then(previewSdkInt => {
 
 ### getReadableVersion()
 
-Gets the application human readable version (same as getVersion() + '.' + getBuildCode())
+Gets the application human readable version (same as getVersion() + '.' + getBuildNumber())
```
Comment (mikehardy) [inline README.md]: This is correct! Thanks for the correction 
https://github.com/react-native-community/react-native-device-info/blob/master/src/index.ts#L322

File: README.md [hunk 3]
```diff
@@ -1139,12 +1139,11 @@ Gets the device User Agent.
 #### Examples
 
 ```js
-DeviceInfo.getUserAgent().then(userAgent => {
-  // iOS: "Mozilla/5.0 (iPhone; CPU iPhone OS 9_1 like Mac OS X) AppleWebKit/601.1.46 (KHTML, like Gecko) Version/9.0 Mobile/13B143"
-  // tvOS: not available
-  // Android: ?
-  // Windows: ?
-});
+let userAgent = DeviceInfo.getUserAgent();
```
Comment (mikehardy) [inline README.md]: Incorrect per the above

**Comment by mikehardy** [comment]
Thanks for the PR! I really appreciate it. I don't think the getUserAgent change is correct but the getReadableVersion one definitely is

File: README.md (hunk 1 above)
Comment (djorkaeffalexandre) [reply README.md]: Oh, my mistake haha, I'm using a older version.. :/ Sorry..
Fixing...

**Comment by djorkaeffalexandre** [comment]
@mikehardy Thanks!!! ❤️

**Comment by mikehardy** [comment]
packing up all the PRs for a release in a moment, this will be out in v5 shortly, thanks again for the help

**Pull request merged**